### ISSUE TITLE
Update to Gradle 8.1.1

### ISSUE THREAD

**Issue by mrcklwhn** [channdara/flutter_localization#6]
After updating to Gradle 8.1.1 the Flutter Localization Plugin doenst work anymore.

My build.gradle: 

```
buildscript {
    ext.kotlin_version = '1.7.20'
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:8.1.1'
        classpath 'com.google.gms:google-services:4.3.10'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}
```

Folowing error: 

A problem occurred configuring project ':flutter_localization'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

Please fix this ASAP.

**Comment by channdara** [comment]
I will check and fix this soon.

**Comment by channdara** [comment]
This issue has been fixed in version **0.1.14**. Please update and check if the problem still exists.
